.eslintignore

```diff
@@ -32,3 +32,6 @@ test/runtime-integration/*/output.js
 test/runtime-integration/*/output-absolute.js
 
 Makefile.js
+
+# TODO: Enable this
+packages/babel-parser/typings
```

Comment:
Can we disable this rule for these files? So other rules are still active for them.

Reply:
I've tried a few things, unfortunately without success, or requiring extensive modifications to the config file.

Reply:
Oh by the way, the error was not caused by this new rule, but that I added the `project` option and tslint warned us that it was not included by tsconfig.

Reply:
Yes. After I add them to the tsconfig.json, both eslint and tsc are happy. So I don't think we have to exclude them, especially when `packages/babel-parser/typings/babel-parser.source.d.ts` is a source file.